save-plugins/warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugin/warn/WarnPlugin.kt

```diff
@@ -155,7 +136,7 @@ class WarnPlugin(
 
         val actualWarningsMap = executionResult.stdout.mapNotNull {
             with(warnPluginConfig) {
-                val line = it.getLineNumber(actualWarningsPattern!!, lineCaptureGroupOut, linePlaceholder!!, null)
+                val line = it.getLineNumber(actualWarningsPattern!!, lineCaptureGroupOut, linePlaceholder!!, null, null, null)
```

Comment:
What are we using this for? Do we need to somehow handle comments in execution output? And how is this result different from just line index?

Reply:
Getting the line number was moved to a separate method so as not to create a separate extractWarning method. It just returns _groups[lineGroupIdx]!!.value.toInt ()_

Reply:
Can we maybe use `getGroupSafe` here too? Passing three `null`s in a method seems odd here.

Reply:
Ok. Fixed